lib/handlebars/compiler/base.js

```diff
@@ -8,7 +8,7 @@ export { parser };
 let yy = {};
 extend(yy, Helpers);
 
-export function parse(input, options) {
+export function parseAST(input, options) {
```

Comment:
I think we should call it "parseWithoutProcessing".

Reply:
Sounds good! Updated.